bin/sce_to_anndata.R

```diff
@@ -121,8 +138,14 @@ sce <- readr::read_rds(opt$input_sce_file)
 # we need this if we have any feature data that we need to add it o
 sample_metadata <- metadata(sce)$sample_metadata
 
-# make main sce czi compliant
-sce <- format_czi(sce)
+# make main sce czi compliant for single objects, or format merged objects
+if (!opt$is_merged) {
+  sce <- format_czi(sce)
+} else {
+  sce <- format_merged_sce(sce)
+}
```

Comment:
remove gratuitous `!`
```suggestion
if (opt$is_merged) {
  sce <- format_merged_sce(sce)
} else {
  sce <- format_czi(sce)
}
```